### ISSUE TITLE
FiFo listener not pulling message again which was not processed 

### ISSUE THREAD

**Issue by Rajneesh-Testsigma** [JaidenAshmore/java-dynamic-sqs-listener#383]
Hi, first  of all thanks for the great library.

I am trying to use FIfoQueuelistener.As from FIFO sqs if any message was processed and deleted from queue, it would visible for other consumer after the expiry of messageVisiblityTimeout.... but in my case it is not appearing. 

Do I need to configure anything else.

**Comment by JaidenAshmore** [comment]
No, it should work as-is. This does sound like a potential problem with your SQS configuration. 

You can see an example FIFO message listener here: [MessageListeners.java#L54](https://github.com/JaidenAshmore/java-dynamic-sqs-listener/blob/f78330a973c87e17fa62136aa26f82610d42e95b/examples/fifo-example/src/main/java/com/jashmore/sqs/examples/MessageListeners.java#L54) where we have examples of it randomly failing to process a message which eventually get picked up again after the timeout.

In this example above we set it up using a local elasticmq server and we set up the receive count and DLQ like so:

```java
attributes.put(
      QueueAttributeName.REDRIVE_POLICY,
      String.format(
          "{\"deadLetterTargetArn\":\"%s\",\"maxReceiveCount\":\"%d\"}",
          deadLetterQueueArn,
          maxReceiveCount
      )
  )
```
see [LocalSqsAsyncClientImpl.java#L21](https://github.com/JaidenAshmore/java-dynamic-sqs-listener/blob/f78330a973c87e17fa62136aa26f82610d42e95b/util/local-sqs-async-client/src/main/java/com/jashmore/sqs/util/LocalSqsAsyncClientImpl.java).

Do you have something similar set up in your SQS queue?
